packages/sdk/src/types/internal/context.ts

```diff
@@ -738,4 +659,19 @@ export class InternalContext {
             });
         }
     }
+
+    public setGrabbability(actorId: string, grabbable: boolean) {
+        const actor = this.actorSet[actorId];
+        if (actor) {
+            actor.created().then(() => {
+                this.protocol.sendPayload({
+                    type: 'update-grabbable',
+                    actorId,
+                    grabbable
+                } as UpdateGrabbable);
+            }).catch((reason: any) => {

```

Comment:
This implementation may change away from a second payload.  See below comment about investigating that within the unity runtime, as behaviors are kind of a special case.

Reply:
This should utilize the actor patch. No need for a new payload type.

---
In reply to: [261903047](https://github.com/Microsoft/mixed-reality-extension-sdk/pull/233#discussion_r261903047) [](ancestors = 261903047)